### PR TITLE
app-laptop/ibam: add X to use-dep for app-admin/gkrellm

### PR DESCRIPTION
**Pull request by laumann**
Closes: https://bugs.gentoo.org/840323
Signed-off-by: Thomas Bracht Laumann Jespersen <t@laumann.xyz>

### CODE REVIEW

File: app-laptop/ibam/ibam-0.5.2-r4.ebuild [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI="6"
```
Comment (arthurzam) [inline app-laptop/ibam/ibam-0.5.2-r4.ebuild:4]: This is a deprecated EAPI!
Please bump it, recommended to EAPI=8 if possible.

(Sorry, but that also will mandate a revbump)

**Review by arthurzam — CHANGES_REQUESTED** [review]
Also, please use `optfeature.eclass` instead of current `pkg_postinst`

File: app-laptop/ibam/ibam-0.5.2-r4.ebuild (hunk 1 above)
Comment (laumann) [reply app-laptop/ibam/ibam-0.5.2-r4.ebuild:4]: thanks @arthurzam I'll just do a revbump to EAPI 8 and fix bugs all-in-one

Comment (laumann) [reply app-laptop/ibam/ibam-0.5.2-r4.ebuild]: done.

**Comment by laumann** [comment]
@arthurzam I think I addressed your comments

File: app-laptop/ibam/ibam-0.5.2-r5.ebuild [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+RDEPEND="
+	gkrellm? (
+		app-admin/gkrellm:2[X]
+		x11-libs/gdk-pixbuf
+		x11-libs/gtk+:2
+	)"
+DEPEND="
+	${RDEPEND}
+	gkrellm? ( virtual/pkgconfig )"
```
Comment (arthurzam) [inline app-laptop/ibam/ibam-0.5.2-r5.ebuild:27]:
```suggestion
DEPEND="${RDEPEND}"
BDEPEND="gkrellm? ( virtual/pkgconfig )"
```

**Review by arthurzam — CHANGES_REQUESTED** [review]
Looks very good! Very good job!

File: app-laptop/ibam/ibam-0.5.2-r5.ebuild (hunk 2 above)
Comment (thesamesam) [reply app-laptop/ibam/ibam-0.5.2-r5.ebuild:27]: I'll fix on merge, thanks!

**Pull request closed without merging**